qemu/tests/cfg/qemu_cpu.cfg

```diff
@@ -18,19 +18,26 @@
     auto_cpu_model = no
 
     # QEMU versions:
-    variants qemu_flavor:
-        - @unknown:
-        - rhel:
-            variants rhel_major_version:
+    variants:
+        - @qemu_flavor_unknown:
+        - qemu_flavor_rhel:
+            variants:
                 - 6:
-                    variants rhel_version:
+                    only Host_RHEL.6
+                    variants:
                         - 6_0:
                         - 6_1:
                         - 6_2:
                         - 6_3:
                         - 6_4:
-        - upstream:
-            variants qemu_version:
+                - 7:
+                    only Host_RHEL.7
+                    variants:
```

Comment:
see alots of such variants but not see where used it?  so any difference about 7_0 and 7_1 ? and any differences between  variants 6 and 7 ?